src/Psalm/ErrorBaseline.php

```diff
@@ -299,7 +299,7 @@ function (array $matches) : string {
                     "\n" .
                     '  php-version="' .
                     "\n    " .
-                    implode("\n    ", explode('&#10;&#9;', $matches[2])) .
```

Comment:
This should consume less resources because str_replace don't need to assign arrays with all the values